Extensions/TextInput/textinputruntimeobject-pixi-renderer.ts

```diff
@@ -297,14 +322,37 @@ namespace gdjs {
       this._input.style.borderWidth = this._object.getBorderWidth() + 'px';
     }
     updateDisabled() {
-      if (!this._input) return;
+      if (!this._form) return;
 
-      this._input.disabled = this._object.isDisabled();
+      this._form.disabled = this._object.isDisabled();
     }
     updateReadOnly() {
+      if (!this._form) return;
+
+      this._form.readOnly = this._object.isReadOnly();
+    }
+
+    updateMaxLength() {
+      const input = this._input;
+      if (!input) return;
+      if (this._object.getMaxLength() <= 0) {
+        input.removeAttribute('maxLength');
+        return;
+      }
+      input.maxLength = this._object.getMaxLength();
+    }
+
+    updatePadding() {
+      if (!this._input) return;
+
+      this._input.style.padding = this._object.getPadding() + 'px';
+    }
+
+    updateTextAlign() {
       if (!this._input) return;
 
-      this._input.readOnly = this._object.isReadOnly();
+      const newTextAlign = this._object.getTextAlign() || 'left';
```

Comment:
Is the `|| 'left'` necessary? I think we can be confident that the object only stores correct values